API v3 is now stable! (#93)
readthedocs/blog: api-v3-stable.rst

.. post:: Feb 1, 2021
   :tags: api, feature, reference
   :author: Manuel
   :location: BCN

> API v3 is now stable!

Reviewer: ```suggestion
API v3 is now stable
```

Not sure about the `!` :)

=====================

> We are excited to announce that our API v3 has reached its stable version,

Reviewer: ```suggestion
We are excited to announce that our API v3 has reached a stable release,
```

and it's now available for all Read the Docs users.
This includes Read the Docs Community and Read the Docs for Business.

> Since we :doc:`announced the beta version </api-v3-beta>` of it,

Reviewer: ```suggestion
Since we :doc:`announced the API v3 beta </api-v3-beta>`,
```

> we have been adding extra functionality and bug-fixing some minor issues based on users feedback.

Reviewer: ```suggestion
we have been adding extra functionality and bug-fixing some minor issues based on user feedback.
```

> The new API v3 *is not* a fully replacement (yet!) of API v2,

Reviewer 2: It feels weird announcing api v3 as stable, but not being a replacement of v2. What are we missing from v2? I think it was just internal usage endpoints.

Author: There are some things that are not migrated yet: `/api/v2/search/`, `/api/v2/embed/`, `/api/v2/footer_html/` and others.

The new API v3 *is not* a fully replacement (yet!) of API v2,

> but **we highly recommend using API v3 for all the new code you write from now on**.

Reviewer: ```suggestion
but **we highly recommend using API v3 for all the new integrations**.
```

> API v2 is planned to be deprecated soon,

Reviewer: ```suggestion
API v2 will be deprecated soon,
```

> though we have not yet set a time frame for deprecation.

Reviewer: ```suggestion
though we don't have a firm timeline for deprecation.
```

> We will alert users with our plans when we do.

Author: I think it's important to communicate this explicitly, but I wasn't sure if the second paragraph was a good place for it. I think it's fine, tho.

I copied the last sentence from our docs to keep consistency: https://docs.readthedocs.io/en/stable/api/v2.html

> Nice features

Reviewer: ```suggestion
New features
```

-------------

> These are some of the possible actions API v3 allows and we like the most,

Reviewer: ```suggestion
We're excited about the following actions that API v3 makes possible:```

- Ability to import new projects
- Activate a version
- Trigger builds
- Check build status
- Manage redirects

> They allow you to easily manage common tasks like setup the documentation for a new project,

Reviewer: What is `They`?

```suggestion
API v3 allow you to easily manage common tasks like setting up documentation for a new project,
```

Author: "They" refers to the features mentioned above :sweat_smile:

> automate the release of a new version, or even pause it if you detect that the build failed for some reason.

Reviewer: ```suggestion
automating the release of a new version,
or pause a release if you detect a failed build.
```

Hook the rename of a file in your VCS to the creation of a redirect in its docs,

> and more!

Reviewer: I feel like this section needs to be written out a bit more. It feels weird to have the bullet points and then also a paragraph with similar content. We should likely remove the bulleted list and talk about a couple common use cases.

Author: My idea here was to highlight some interesting features (to make the reader match them into their own use cases) and then combine those features into useful real-life use cases we already known. Makes sense? Any suggestion about how I can achieve that without sounding repetitive here?

Reviewer: It is basically just another list, so we should either make it a list, or fully explain the use cases. I'll push up an update making it a list, since that seems better for now.

If you want to know more about it,
please check out the full `APIv3 documentation <https://docs.readthedocs.io/page/api/v3.html>`_.

> Help us to make it better

Reviewer: ```suggestion
Help us make it better
```

-------------------------

We always love to hear from users about how they are using the API.

> We want to help users to manage their projects' documentation in the easiest and less repetitive way possible.

Reviewer: ```suggestion
We want to help you manage your documentation in the easiest and most efficient way possible.
```
